### DISCUSSION TITLE
how to add custom domain in chatwoot help center

### DISCUSSION THREAD

**Discussion by neyunse** [coollabsio/coolify#3450] [Q&A]
Hi, I installed chatwoot and am having trouble adding a custom domain to the help portal. According to the chatwoot documentation I should put in my DNS [subdomain].x.com, however, when accessing from hc.x.com I get the 404 from coolify; any idea how to place the help center in a custom domain or subdomain? [https://www.chatwoot.com/docs/self-hosted/configuration/help-center](https://www.chatwoot.com/docs/self-hosted/configuration/help-center)

**Comment by neyunse** [comment]
Update: I tried with CF tunels and caddy and I have the same problem, it seems that coolify is not allowing access to the help center custom domain in chatwoot (Self-Hosted) or the domain is never configured.

**Reply by bitalicious** [reply]
Could you solve it? I am having exact same problem, except that I am not using coolify, but cloudron.

**Reply by neyunse** [reply]
> Could you solve it? I am having exact same problem, except that I am not using coolify, but cloudron.

I Just add http://help.example.com:(port) and it worked. Of course, after many tries.

"http://chatwoot.example.com:(port), http://help.example.com:(port)" Although I'm not sure if you first have to configure the URL in Coolify and then in the chatwoot instance

**Reply by robin-scharf** [reply]
Just adding here, I use this "http://chatwoot.example.com:(port), http://help.example.com:(port)" and it worked for me as well. I have chat.example.com:3000 and docs.example.com:3000
